imap_processing/codice/l0/__init__.py

```diff
@@ -0,0 +1,515 @@
+LOSSY_A_TABLE = {
```

Comment:
Might want to put this in a separate file to make it more explicit what these are referring to? `constants.py` or `table_data.py`